channels/gr.m3u

```diff
@@ -241,3 +241,7 @@ http://5db313b643fd8.streamlock.net:1935/ZerounoTV/ZerounoTV/playlist.m3u8
 https://zouglalive-lh.akamaihd.net/i/zouglalive_1@340792/master.m3u8
 #EXTINF:-1 tvg-id="Krete TV GR" tvg-name="Krete TV GR" tvg-country="GR" tvg-language="Modern Greek (1453-)" tvg-logo="https://i.imgur.com/Ldt07Lh.png" group-title="",ΚρήτηTV (720p)
 http://live.cretetv.gr:1935/cretetv/myStream/f1tv.m3u8
+#EXTINF:-1 tvg-id="ANT1CYPRUS.GR" tvg-name="ANT1 Cyprus" tvg-country="CY" tvg-language="Modern Greek (1453-)" tvg-logo="https://www.worldtvpc.com/onlinevideosites/img/ant1.jpg" group-title="",ANT1 Cyprus
+https://www.ant1.com.cy/ajax.aspx?m=Atcom.Sites.Ant1iwo.Modules.TokenGenerator&videoURL=https://l2.cloudskep.com/antl2/abr/playlist.m3u8
```

Comment:
I don't think that one will work, because it's a text that says where the link is. Few seconds.

Reply:
Works in VLC for me

Reply:
Interesting. Wouldn't predict that would work lol. Well then, here's an alternative one in case of (just redirects in the URL inside the link : https://iptv-all.lanesh4d0w.codes/cyprus/ant1

Reply:
RIK S and RIK SAT are the same channels. Duplicate entries.

Reply:
ANT1cy and VOULIcy should be under cy.m3u

Reply:
We're referencing lyngsat to determine an origin country. According to them those channels have headquarters in Greece. Those will be present in Cyprus playlist tho

Reply:
Oh okay thanks for letting me know, wasn't aware of that. There is also an outdated link in lyngsat for VOULI cy that is not working. Latest link for ANT1cy is not playing under VLC. Line 226/227 in gr.m3u

Reply:
They're changing servers I suppose. Let me see.

Reply:
It seems it's the same server. I haven't made a mistake when I've made the workaround either.

Maybe there's a specific user-agent to use, since it works on computer. @Dum4G knows this aspect more than I do.

Reply:
@Dum4G Try my link with referer positioned in https://www.ant1.com.cy/

Reply:
Didn't work

Reply:
Interesting... there's something in this link that blocks then.

Reply:
Just made Alpha Cyprus, pushing it soon onto my server. Works perfectly on my watch.

Testing a plan B for Ant1.

Reply:
OG link to a function is no longer providing working link. Maybe needs some proper orgin or has become geo-blocked

Reply:
It's geoblocked in its root. But even I with a Cyprus IP can only watch it using the player. Pretty sure there's a sort of referrer or origin lock.

Reply:
Something sounds weird.

![image](https://user-images.githubusercontent.com/30985701/128088051-8ced492d-63a2-45c5-86be-1abf05a88d29.png)

Reply:
Found the issue, pushing the changes.

Reply:
Ant1 is the same link as the one i've sent above.
Alpha : https://iptv-all.lanesh4d0w.codes/cyprus/alpha-tv

Reply:
Omega TV : https://iptv-all.lanesh4d0w.codes/cyprus/omega-tv
Vouli (in case the link in list goes down) : https://iptv-all.lanesh4d0w.codes/cyprus/vouli

Reply:
It seems that all the Cloudskep links cease to work once I push them on my server, but works perfectly when being put on my server. Might be because they're geo-locked even at that scale... Dang it.